tests/ci/config.go

```diff
@@ -0,0 +1,51 @@
+package ci
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	CON "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+)
+
+var RHCS = new(RHCSconfig)
+
+// RHCSConfig contains platforms info for the RHCS testing
+type RHCSconfig struct {
+	// Env is the OpenShift Cluster Management environment used to provision clusters.
+	RHCSEnv           string `env:"RHCS_ENV" default:"staging" yaml:"env"`
```

Comment:
How would I change it to another environment? with environment variable from PROW CI job?

Reply:
yeah with the `CON.OCMEnv` var declared